pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "cookiecutters"
+version = "0.1.0"
+description = "MobileCoin cookiecutter templates"
+authors = ["MobileCoin"]
+license = "Apache-2.0"
```

Comment:
Poetry assumes a package will be published, I don't think we'll be doing that, but left this here anyway